src/polyfill.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
can we just put this into a different directory? i personally dont like having something like this unimportant in the "root"-directory - what do you think?

Reply:
We could put our version of 
`\Symfony\Component\Console\Output\ConsoleSectionOutput` and this polyfill.php into the same directory.

like: `src/Polyfills/auto-polyfill.php`

Reply:
splendid idea